pkg/transformer/utils.go

```diff
@@ -141,3 +143,80 @@ func Print(name, path string, trailing string, data []byte, toStdout, generateJS
 	}
 	return file, nil
 }
+
+// GetComposeFileDir returns compose file directory
+func GetComposeFileDir(inputFiles []string) (string, error) {
+	// Lets assume all the docker-compose files are in the same directory
+	inputFile := inputFiles[0]
+	if strings.Index(inputFile, "/") != 0 {
+		workDir, err := os.Getwd()
+		if err != nil {
+			return "", err
+		}
+		inputFile = filepath.Join(workDir, inputFile)
+	}
+	log.Debugf("Compose file dir: %s", filepath.Dir(inputFile))
+	return filepath.Dir(inputFile), nil
+}
+
+func BuildDockerImage(service kobject.ServiceConfig, name string, relativePath string) error {
```

Comment:
I see the results of `-v`, still it is shown only when build is done. How about we show it in real time, that was my original concern!

Being curious: what is problem in spawing another go routine?